README.md

```diff
@@ -207,8 +207,10 @@ Copyright &copy; 2020 [Mineiros GmbH][homepage]
 [badge-terraform]: https://img.shields.io/badge/terraform-0.13%20and%200.12.20+-623CE4.svg?logo=terraform
 [badge-slack]: https://img.shields.io/badge/slack-@mineiros--community-f32752.svg?logo=slack
 
+<!-- markdown-link-check-disable -->
 [build-status]: https://mineiros.semaphoreci.com/projects/terraform-github-team
```

Comment:
the project is public in semaphore so we can indeed have it checked by markdown-link-checker.